### DISCUSSION TITLE
Install arbitrary docker app via a link to yaml file

### DISCUSSION THREAD

**Discussion by mehrdadfeller** [ubopod/ubo_app#81] [Ideas]
So far the end user can install dockers apps from a list of offered containers out of the box. Alternatively they can install portainer and use that to install other docker containers. 

I believe we need to add a third way to seamlessly fetch and install a docker container by scanning a QR code that links to a yaml config file. These can be served on a Github gist or any endpoint. The user should include all necessary config params and credentials in their yaml file. 

Let's discuss the details here and work out the specs.

**Comment by zytegalaxy** [comment]
This is now supported in release version `1.3.0`
